bigtop-bigpetstore/bigpetstore-transaction-queue/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin:'application'
+mainClassName = "org.apache.bigtop.load.LoadGen"
+
+sourceCompatibility = 1.7
+version = '1.0'
+
+// Begin fat jar reuirements..
+buildscript {
+        repositories {
+            jcenter()
+        }
+
+        dependencies {
+            classpath 'eu.appsatori:gradle-fatjar-plugin:0.3'
+        }
+}
+apply plugin: 'eu.appsatori.fatjar'
+//end fat jar reuirements..
+
+fatJar {
+        exclude 'META-INF/*.DSA'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.fasterxml:jackson-module-json-org:0.9.1'
+    compile 'org.apache.commons:commons-io:1.3.2'
+    compile 'org.apache.httpcomponents:httpclient:4.4'
+    compile group:'org.apache.commons', name:'commons-lang3',version:'3.3.2'
+    compile files('libs/bigpetstore-data-generator-0.2.jar')
```

Comment:
We should pull the jar from bintray.  Look at the BPS Spark gradle file for directions on how to do this.  Make sure to use the 0.2.1.jar not, 0.2.  Important bug fix in there.